lumigator/infra/mzai/helm/lumigator/values.yaml

```diff
@@ -22,7 +22,6 @@ rayWorkerGPUs: ""
 
 serviceAccountName: ""
 
-summarizerWorkDir: ""
```

Comment:
@chainlink anything else that needs to be changed on the infra side?

Reply:
Looks good!